app/controllers/users_controller.rb

```diff
@@ -345,6 +345,16 @@ def shortlink
     end
   end
 
+  def verify_email
+    @user = User.find(params[:id])
+    if @user.validate_token(params[:token])
+      puts "Email successfully validated"
```

Comment:
It would be better to flash a message on the login page when user gets redirected instead of output in console.